src/GraphQL/Types/Scalars/ScalarGraphType.cs

```diff
@@ -152,5 +165,25 @@ public virtual IValue ToAST(object value)
                 return null;
             }
         }
+
+        protected internal object ThrowLiteralConversionError(IValue input)
+        {
+            throw new ArgumentException($"Unable to convert '{input}' to '{Name}'");
+        }
+
+        protected internal IValue ThrowASTConversionError(object value)
+        {
+            throw new InvalidOperationException($"Unable to convert '{value}' of the scalar type '{Name}' to an AST representation.");
```

Comment:
```suggestion
            throw new InvalidOperationException($"Unable to convert '{value}' to an AST representation for the scalar type '{Name}'.");
```

Reply:
I tried to make the messages similar for all four methods.

Reply:
👍

Once I can get existing tests to pass again, then I'll commit the revised exception messages.

Reply:
Code was unused when done (since ToAST default implementation was sufficient in every case), so I removed the method.

Reply:
Already resolved.